docs/changes.rst

```diff
@@ -1,4 +1,4 @@
-.. include:: _authors.rst
+ds.. include:: _authors.rst
```

Comment:
Is this addition on purpose?

Reply:
Of course not 😮  I even couldn't catch it on the github changes view... Thanks!